hulks/base.py

```diff
@@ -26,8 +28,16 @@ def handle(self, args=None):
         return int(not retval)
 
     def lines_iterator(self, filename):
-        with open(filename) as fp:
-            for line_number, line in enumerate(fp.readlines(), 1):
-                # heuristic, so we dont need to handle all "comment" syntax accross languages
-                if ' noqa' not in line.lower():
-                    yield line_number, line
+        if not mimetypes.guess_type(filename)[0].startswith('text/'):
```

Comment:
better not use this, there's probably mimetypes that doesn't start with `text/` that would trigger false positives here (like `application/javascript` is a valid mimetype)

Reply:
I am just removing this test altogether them - Noboy seems to have a strong opinion about it (neither me), and checking the mimetype.... call now, it is rather unreliable for other files as well (it can even return just (None, None)  instead of (str, None)   for a lot of filenames.)